docs/tpm2-pkcs11_object_auth_model.md

```diff
@@ -0,0 +1,528 @@
+# tpm2-pkcs11 object & authorization model
+
+## Introduction
+
+The objective of this document is to present an architectural overview of the
+tpm2-pkcs11 object and authorization model implemented in the design. The topics
```

Comment:
not implemented yet. I don't want people to think this is in place now. So **will be implemented** and bold it.